### PR TITLE
Feature/delayed

### PR DESCRIPTION
**Pull request by lucblender**
Minor:
- Added supportedFormat in Endpoint Model
- Added version in Endpoint 

Add delayed message:
- Created Object Structure
- Added service handling them, due to database migration, I didn't complete the logic of this service, only the parsing of the message

### CODE REVIEW

File: src/main/kotlin/ch/hevs/cloudio/cloud/model/Endpoint.kt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package ch.hevs.cloudio.cloud.model
 
 data class Endpoint(
+        val version: String = "",
+        val supportedFormat: Set<String> = mutableSetOf(),
```
Comment (cm0x4D) [inline src/main/kotlin/ch/hevs/cloudio/cloud/model/Endpoint.kt]: supportedFormats

File: src/main/kotlin/ch/hevs/cloudio/cloud/services/DelayedService.kt [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            } else {
+                log.error("Unrecognized message format in @delayed message from $endpointId")
+            }
+        } catch (exception: Exception) {
+            log.error("Exception during @delayed message handling:", exception)
+        }
+    }
+
+    fun handleUpdate(topic: String, attribute: Attribute){
+        //TODO
```
Comment (cm0x4D) [inline src/main/kotlin/ch/hevs/cloudio/cloud/services/DelayedService.kt:99]: Is this intentionally not implemented?

Maybe add a log warn message to simplify testing.

**Review by cm0x4D — CHANGES_REQUESTED** [review]

File: src/main/kotlin/ch/hevs/cloudio/cloud/services/DelayedService.kt (hunk 2 above)
Comment (lucblender) [reply src/main/kotlin/ch/hevs/cloudio/cloud/services/DelayedService.kt:99]: from my PR message:
> Added service handling them, due to database migration, I didn't complete the logic of this service, only the parsing of the message

Comment (cm0x4D) [reply src/main/kotlin/ch/hevs/cloudio/cloud/services/DelayedService.kt:99]: Ok, makes sense then.

**Review by cm0x4D — APPROVED** [review]

**Review by cm0x4D — APPROVED** [review]

**Pull request merged**